packages/dds/cell/README.md

```diff
@@ -2,24 +2,10 @@
 
 The `SharedCell` Distributed Data Structure (DDS) stores a single, shared value that can be edited or deleted.
 
-<!-- AUTO-GENERATED-CONTENT:START (README_INSTALLATION_SECTION:includeHeading=TRUE) -->
+This package is primarily intended as a minimal example of how distributed data structures work.
+For real-world scenarios, we recommend using one of our other DDSs, like [SharedTree](https://fluidframework.com./docs/data-structures/tree/).
 
-<!-- prettier-ignore-start -->
```

Comment:
Just curious but is it fine for these autogenerated sections to be manually deleted? Is there some markdown-magic config or setting that includes/excludes generating content for this file?

Reply:
This README was set up to use a bunch of individual markdown-magic macros, each of which has those ignore comments inside. This PR changed the README to use our all-in-one template that includes all of the common README contents in one, so there is a single ignore block wrapping all of the contents instead of per-section.